test/Riok.Mapperly.Tests/_snapshots/EnumerableTest.ShouldUpgradeNullabilityInDisabledNullableContextInSelectClause#Mapper.g.verified.cs

```diff
@@ -26,12 +26,12 @@ public partial class Mapper
 
     private D[] MapToDArray(C[] source)
     {
-        var target = new D[source.Length];
+        var target = new D?[source.Length];
```

Comment:
While the target is now correctly created, the nullability of the return type is incorrect.

The nullability of the `C[]` type is also incorrect (should be `C?[]`). This could also be solved in a separate PR, since an issue for this is still open

Reply:
Should be resolved in https://github.com/riok/mapperly/issues/229